Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml

```diff
@@ -393,3 +393,17 @@
   metamorphicSprite:
     sprite: Objects/Consumable/Drinks/the_martinez.rsi
     state: icon
+
+- type: reagent
+  id: WhiteGilgamesh
```

Comment:
Should this not be in the alcohol.ftl instead, because it's made from beer?

Reply:
it's not alcoholic, but i should probably make it that